backend/siarnaq/api/compete/views.py

```diff
@@ -387,6 +388,80 @@ def scrimmage(self, request, pk=None, *, episode_id):
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    def get_historical_rating_ranking(self, episode_id, queryset, limit=None):
+        """
+        Retrieve historical ratings for teams in a specific episode, sorted by
+        highest rating.
+
+        This function processes a set of matches to calculate and return historical
+        ratings for teams participating in a given episode. The results are ordered
+        by the maximum rating achieved, in descending order.
```

Comment:
previously were we also doing the ranking on maximum rating ever achieved?
(i commented this in slack too)

Reply:
Good catch!